index.js

```diff
@@ -18,6 +18,12 @@ module.exports = function(env, callback) {
       nenv.addFilter(name, filter);
     });
   }
+  if(env.config.nunjucks && env.config.nunjucks.filtermodules) {
+    env.config.nunjucks.filtermodules.map( function (name) {
+      var filter = require(name);
+      filter.install(nenv);
```

Comment:
I'm pretty new to wintersmith/nunjucks, was just playing around, so I'd let you decide if you want to pull this in and who should be informed to document this in the API, because I don't know this ecosystem well.

I added a note about the required `install`-method.